### PR TITLE
Fixed BigQuery and Logging tests

### PR DESCRIPTION
**Pull request by tmatsuo**
Also fixed the coverage configs

### CODE REVIEW

**Comment by tmatsuo** [comment]
Oops, the BigQuery test is failing. I'll look into it

File: bigquery/api/src/QueryCommand.php [hunk 1]
```diff
@@ -90,18 +90,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 throw new Exception('You must supply a query argument');
             }
         }
+        var_dump($input->getOption('standard-sql'));
```
Comment (bshaffer) [inline bigquery/api/src/QueryCommand.php]: 👎

File: testing/run_test_suite.sh [hunk 2]
```diff
@@ -20,8 +20,8 @@ if [ "${RUN_CS_FIXER}" = "true" ]; then
     ${HOME}/php-cs-fixer fix --dry-run --diff
 fi
 
-# loop through all directories containing "phpunit.xml" and run them
-find * -name phpunit.xml -not -path '*/vendor/*' -exec dirname {} \; | while read DIR
+# loop through all directories containing "phpunit.xml*" and run them
+find * -name 'phpunit.xml*' -not -path '*/vendor/*' -exec dirname {} \; | while read DIR
```
Comment (bshaffer) [inline testing/run_test_suite.sh]: 👍

**Comment by bshaffer** [comment]
remove the `var_dump` and this is LGTM

**Pull request merged**